rest/class-boldgrid-backup-rest-test.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * File: class-boldgrid-backup-rest-test.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      X.X.X
```

Comment:
you're using X.X.X in this file instead of SINCEVERSION ..... Just putting this here for ya so these don't get missed when you do your search and replace later

Reply:
Nice catch https://github.com/BoldGrid/boldgrid-backup/commit/cc4f441ae5d3cebf2bc8c0c3b1c682613e6007d9